### PR TITLE
add release 1.36 post

### PR DESCRIPTION
**Pull request by solvingj**

### CODE REVIEW

File: _posts/2021-05-10-New-conan-release-1-36.markdown [hunk 1]
```diff
@@ -0,0 +1,271 @@
+---
+layout: post 
+comments: false 
+title: "Conan 1.36 : New cpp_info property system, Multiple CMake enhancements,
+Support build_requires testing in test_package, New –-build exclude syntax"
```
Comment (memsharded) [inline _posts/2021-05-10-New-conan-release-1-36.markdown:5]:
```suggestion
Support build_requires testing in test_package, New --build exclude syntax" 
```

**Review by danimtb — APPROVED** [review]

File: _posts/2021-05-10-New-conan-release-1-36.markdown [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+it if appropriate.
+
+In summary, this strategy makes the recipe more generic and less coupled to
+supporting specific generators, and puts the onus on the recipe authors to
+support known properties. Some built-in properties are already defined and used
+in current generators, but this strategy also supports completely arbitrary
+properties for custom-recipe and custom-generator use-cases. For that reason, we
+think this feature will see extensive use in enterprise package environments.
+
+## CMakeToolchain Enhancements
```
Comment (memsharded) [inline _posts/2021-05-10-New-conan-release-1-36.markdown:88]: Call it "new CMake integration enhancements"

Comment (memsharded) [reply _posts/2021-05-10-New-conan-release-1-36.markdown:88]: Make this the first item in the blog post.

File: _posts/2021-05-10-New-conan-release-1-36.markdown [hunk 3]
```diff
@@ -0,0 +1,271 @@
...
+
+One of Conan's most novel features among package managers in general is the
+`--build` flag. When calling `conan create` or `conan install`, callers can
+specify precisely which packages in the graph they want to build from source,
+and which ones they can use precompiled binaries for. Users had the following
+syntactic options:
+
+* `--build=all` or `--build`: all packages
+* `--build=package_name` : a single specific package by name
+* `--build=package_1,package_2` : a list of specific packages
```
Comment (memsharded) [inline _posts/2021-05-10-New-conan-release-1-36.markdown:240]: I don't think this is an option, you need to do ``--build=package1 --build=package2``

**Review by memsharded — CHANGES_REQUESTED** [review]

**Pull request merged**